src/App.js

```diff
@@ -10,7 +10,7 @@ function App() {
 
 
   const getData = async ()  => {
-    const res = await axios.get("https://api.coingecko.com/api/v3/coins/markets?vs_currency=usd&order=market_cap_desc&per_page=100&page=1");
+    const res = await axios.get("https://api.coingecko.com/api/v3/coins/markets?vs_currency=USD&order=market_cap_desc&per_page=5&page=1");
```

Comment:
Just make sure that the list don't change

Reply:
done